instances/main.tf

```diff
@@ -68,16 +70,18 @@ module "nano-x64-minimal-pool" {
   }
 }
 
+## arm64
+
 module "arm-build-pool" {
   source = "./azp-build-asg"
 
-  ami_prefix           = "envoy-azp-arm64"
+  ami_prefix           = "envoy-azp-build-arm64"
   aws_account_id       = "457956385456"
   azp_pool_name        = "envoy-arm-large"
   azp_token            = var.azp_token
   disk_size_gb         = 1000
   idle_instances_count = 1
-  instance_types       = ["r6g.8xlarge"]
+  instance_types       = ["m6g.16xlarge"]
```

Comment:
this was required as the r6g.8xlarge doesnt seem to be available for jammy-minimal base - not sure if we can build a different ami or find a better vm

this speeds up the arm job, just testing how much by, so not sure if we can justify keeping this